include/rocksdb/listener.h

```diff
@@ -194,10 +194,32 @@ enum class BackgroundErrorReason {
   kManifestWriteNoWAL,
 };
 
+enum class WriteStallCause {
+  kMemtableLimit,
+  kWriteBufferManagerLimit,
+  kL0FileCountLimit,
+  kPendingCompactionBytes,
+  // Add new WriteStallCause before `kNone`
+  kNone,
+};
+
 enum class WriteStallCondition {
-  kNormal,
   kDelayed,
   kStopped,
+  // Add new WriteStallCondition before `kNormal`
+  kNormal,
+};
+
+struct CFWriteStallStatsMapKeys {
```

Comment:
Would it be too much if we create a .h in include just for this class and WriteStallCause, WriteStallCondition?